community.main/pom.xml

```diff
@@ -0,0 +1,50 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.kaazing</groupId>
+        <artifactId>community.parent</artifactId>
+        <version>develop-SNAPSHOT</version>
+    </parent>
+
+    <groupId>org.kaazing</groupId>
+    <artifactId>community</artifactId>
+    <packaging>pom</packaging>
+
+    <name>Community</name>
+    <url>https://github.com/kaazing/community</url>
+
+    <description>Community POM for all Maven projects</description>
+
+    <scm>
+        <url>https://github.com/kaazing/community</url>
+        <connection>scm:git:https://github.com/kaazing/community.git</connection>
+        <developerConnection>${project.scm.connection}</developerConnection>
+    </scm>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.kaazing</groupId>
+            <artifactId>community.license</artifactId>
+            <!-- We really want -->
+            <!--     <version>{project.version}</version> -->
+            <!-- but due to inheritance rules, this won't work. -->
+            <version>RELEASE</version>
```

Comment:
I see: presumably for this [reason](http://stackoverflow.com/questions/30571/how-do-i-tell-maven-to-use-the-latest-version-of-a-dependency)